  Header header
  uint32 message_id
  RotationsAtRpm[10] motor
